package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@myuw-web-components/mwc-site",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Now that we are adding a new file (changelog) to the repository, I would suggest bumping version to 1.1.0.

Reply:
will do! thanks

Reply:
I think too that 1.0.0 is fine, let's keep it that way?